src/Analysis/Ast/Impl/Types/PythonClassType.Generics.cs

```diff
@@ -331,24 +331,24 @@ internal partial class PythonClassType {
         /// Transfers members from generic class to the specific class type
         /// while instantiating specific types for the members.
         /// </summary>
-        private void SetClassMembers(PythonClassType classType, IArgumentSet args) {
+        private void SetClassMembers(IArgumentSet args) {
             // Add members from the template class (this one).
             // Members must be clones rather than references since
             // we are going to set specific types on them.
-            classType.AddMembers(this, true);
+            AddMembers(this, true);
```

Comment:
The meaning of `this` is changing here, since before `this` meant the parent type, while now it means the child. Similar to the other one above.

Reply:
The comment here would need to be modified too.

Reply:
It actually needs to pass the template class